crates/air/Cargo.toml

```diff
@@ -12,14 +12,22 @@ repository.workspace = true
 rust-version.workspace = true
 
 [dependencies]
+air_fs = { workspace = true }
 air_r_formatter = { workspace = true }
 air_r_parser = { workspace = true }
 anyhow = { workspace = true }
+biome_console = { workspace = true }
+biome_diagnostics = { workspace = true }
 biome_formatter = { workspace = true }
-clap = { workspace = true }
+biome_parser = { workspace = true }
+clap = { workspace = true, features = ["wrap_help"] }
```

Comment:
The magic `wrap_help` here enables auto wrapping of help text in the cli
https://github.com/astral-sh/ruff/pull/9633